frontend/api/api.ts

```diff
@@ -55,7 +55,7 @@ export class Api {
       method: "GET",
       headers: {
         Accept: "application/json",
-        Authorization: `Bearer: ${btoa(token)}`,
+        Authorization: `Bearer ${token}`,
```

Comment:
Does this work? I read over the RFC for the authorization header and I thought it assumed that the Bearer token was base64 encoded?

Reply:
I tried it both ways and ran into an error when it was `btoa(token)` but it works like this

Reply:
Huh, that's confusing. Maybe Fetch does some magic under the hood I just didn't know about. Thanks for catching this